### ISSUE TITLE
Question: Is the 3 partitions in the processed folder correspond to train/valid/test? 

### ISSUE THREAD

**Issue by PraljakReps** [petergroth/FLOP#4]
Do the sequences associated with `df['part_2'] == 1` correspond to the test set, while those linked with `df['part_0'] == 1` and `df['part_1'] == 1` represent the training and validation sets, respectively?

**Comment by petergroth** [comment]
For small datasets, especially the GH114 family of sequences, cross-validation is the recommended validation technique. We evaluated the three datasets this way, whereby all partitions appear as training, validation, and test partitions for a total of six combinations.
